src/wallet/rpcwallet.cpp

```diff
@@ -3866,6 +3871,185 @@ static UniValue listlabels(const JSONRPCRequest& request)
     return ret;
 }
 
+static RPCHelpMan send()
+{
+    return RPCHelpMan{"send",
+        "\nSend a transaction.\n",
+        {
+            {"outputs", RPCArg::Type::ARR, RPCArg::Optional::NO, "a json array with outputs (key-value pairs), where none of the keys are duplicated.\n"
+                    "That is, each address can only appear once and there can only be one 'data' object.\n"
+                    "For convenience, a dictionary, which holds the key-value pairs directly, is also accepted.",
+                {
+                    {"", RPCArg::Type::OBJ, RPCArg::Optional::OMITTED, "",
+                        {
+                            {"address", RPCArg::Type::AMOUNT, RPCArg::Optional::NO, "A key-value pair. The key (string) is the bitcoin address, the value (float or string) is the amount in " + CURRENCY_UNIT + ""},
+                        },
+                        },
+                    {"", RPCArg::Type::OBJ, RPCArg::Optional::OMITTED, "",
+                        {
+                            {"data", RPCArg::Type::STR_HEX, RPCArg::Optional::NO, "A key-value pair. The key must be \"data\", the value is hex-encoded data"},
```

Comment:
`RPCArg::Type::STR_HEX` is not a key-value pair either. I think you mean `RPCArg::Type::OBJ` here too.

Reply:
I think the type is correct, see also `walletcreatefundedpsbt`, but it's confusing (in the code) that the wrapping object is described next to the `data` field. cc @achow101 / @MarcoFalke thoughts?

Reply:
The type is correct because it is describing the type of the value in this key-value pair. The weirdness is that this output specification is the almost the only place where the key is not fixed but user defined.